packages/manager/modules/pci/src/projects/project/data-processing/notebooks/notebook-details/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,24 @@
+{
+  "data_processing_notebooks_details_dashboard_label": "Tableau de bord",
+  "data_processing_notebooks_details_information_label": "Informations générales",
+  "data_processing_notebooks_details_information_editor_label": "Live-code editor",
+  "data_processing_notebooks_details_information_name_label": "Nom",
+  "data_processing_notebooks_details_information_region_label": "Région",
+  "data_processing_notebooks_details_information_framework_label": "Framework",
+  "data_processing_notebooks_details_information_privacy_label": "Confidentialité",
+  "data_processing_notebooks_details_information_privacy_public_label": "Public",
+  "data_processing_notebooks_details_information_privacy_private_label": "Privé",
+  "data_processing_notebooks_details_lifecycle_label": "Lifecycle",
+  "data_processing_notebooks_details_lifecycle_status_label": "Statut",
+  "data_processing_notebooks_details_lifecycle_start_label": "Démarrer",
+  "data_processing_notebooks_details_lifecycle_stop_label": "Arrêter",
+  "data_processing_notebooks_details_lifecycle_delete_label": "Supprimer",
+  "data_processing_notebooks_details_lifecycle_last_start_date_label": "Démarré depuis le",
+  "data_processing_notebooks_details_lifecycle_elapsed_label": "Temps écoulé",
+  "data_processing_notebooks_details_resources_label": "Resources",
+  "data_processing_notebooks_details_resources_notebook_label": "Notebook",
+  "data_processing_notebooks_details_resources_notebook_NB1-1_label": "<strong>NB1-1 :</strong> 1 vCore CPU, 4 Gio memory.",
```

Comment:
**nitpick:** `NB1-1 ` can be maintained as a constant and passed using `translate-values`. Thus avoiding html in translation files `<strong>`

Reply:
Fixed :)